pkgs/top-level/all-packages.nix

```diff
@@ -32484,7 +32484,9 @@ with pkgs;
 
   khard = callPackage ../applications/misc/khard { };
 
-  kid3 = libsForQt5.callPackage ../applications/audio/kid3 { };
+  kid3 = qt6Packages.callPackage ../applications/audio/kid3 { };
+
+  kid3-qt = qt6Packages.callPackage ../applications/audio/kid3 { withKDE = false; };
```

Comment:
Any particular reason why? kid3 is an official KDE application and with KDE being our default DE, I would imagine that most people would want the KDE version.

Reply:
If  the main version with KDE integration  is called `kid3`, then it's natural to call the other `kid3-no-kde`.

Further, calling the second one `kid3-qt` gives me the misleading impression the other one has no Qt dependencies.

Reply:
I get your point, it's just that the names typically show what is included in case of multiple variations and typically not what is *not* included.

`kid3` also has a cli option, so if we wanted to make it *really* obvious, we could do 3:

```
kid3-cli
kid3-kde
kid3-qt
```

And then do `kid3 = kid3-kde`;